index.d.ts

```diff
@@ -10,9 +10,10 @@ declare module "fhirpath" {
     fhirData: any,
     path: string | Path,
     context: Context,
-    model?: Model,
+    model: Model,
```

Comment:
The `model` parameter should be optional here and in "src/fhirpath.js".
The `context` parameter should also be optional here and there.
Thank you for bringing this to my attention!

Reply:
For "src/fhirpath.js" I mean update the JSDoc:
`@param {object} context` should be replaced with `@param {object} [context]`
`@param {object} model` should be replaced with `@param {object} [model]`